src/core/init/types.ts

```diff
@@ -57,6 +57,7 @@ import {
 /** Object awaited by the `Init` on each clock tick. */
 export interface IInitClockTick { position : number;
                                   getCurrentTime : () => number;
+                                  mediaInternalSeeking: boolean;
```

Comment:
Well, `Init` doesn't really care about that property, no?

We are not obligated to define it as a needed property for the `Init`'s clock tick